[2023-04-13T20:29:48.949+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.inject_data_Zhmerynka manual__2023-04-13T20:29:34.713243+00:00 [queued]>
[2023-04-13T20:29:48.971+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.inject_data_Zhmerynka manual__2023-04-13T20:29:34.713243+00:00 [queued]>
[2023-04-13T20:29:48.972+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-13T20:29:48.974+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-13T20:29:48.975+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-13T20:29:48.999+0000] {taskinstance.py:1309} INFO - Executing <Task(SqliteOperator): inject_data_Zhmerynka> on 2023-04-13 20:29:34.713243+00:00
[2023-04-13T20:29:49.007+0000] {standard_task_runner.py:55} INFO - Started process 7041 to run task
[2023-04-13T20:29:49.014+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'inject_data_Zhmerynka', 'manual__2023-04-13T20:29:34.713243+00:00', '--job-id', '346', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpvmszix9f']
[2023-04-13T20:29:49.018+0000] {standard_task_runner.py:83} INFO - Job 346: Subtask inject_data_Zhmerynka
[2023-04-13T20:29:49.120+0000] {task_command.py:389} INFO - Running <TaskInstance: weather_dag.inject_data_Zhmerynka manual__2023-04-13T20:29:34.713243+00:00 [running]> on host ffea11fb791c
[2023-04-13T20:29:49.350+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_dag
AIRFLOW_CTX_TASK_ID=inject_data_Zhmerynka
AIRFLOW_CTX_EXECUTION_DATE=2023-04-13T20:29:34.713243+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-13T20:29:34.713243+00:00
[2023-04-13T20:29:49.356+0000] {sql.py:254} INFO - Executing: 
                INSERT INTO measures_ext (city, timestamp, temp, humidity, cloudiness, wind_speed) VALUES
                    ("Zhmerynka",
                    1681417779,
                    281.24,
                    89,
                    "overcast clouds",
                    1.61);
            
[2023-04-13T20:29:49.380+0000] {base.py:73} INFO - Using connection ID 'measure_db' for task execution.
[2023-04-13T20:29:49.756+0000] {base.py:73} INFO - Using connection ID 'measure_db' for task execution.
[2023-04-13T20:29:49.763+0000] {sql.py:375} INFO - Running statement: 
                INSERT INTO measures_ext (city, timestamp, temp, humidity, cloudiness, wind_speed) VALUES
                    ("Zhmerynka",
                    1681417779,
                    281.24,
                    89,
                    "overcast clouds",
                    1.61);
            , parameters: None
[2023-04-13T20:29:49.781+0000] {sql.py:384} INFO - Rows affected: 1
[2023-04-13T20:29:49.815+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=inject_data_Zhmerynka, execution_date=20230413T202934, start_date=20230413T202948, end_date=20230413T202949
[2023-04-13T20:29:49.880+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-13T20:29:49.907+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
